scripts/utils/index.ts

```diff
@@ -43,8 +35,26 @@ export enum DEPLOYMENT_SALTS {
   WALLET_FACTORY = "DEVX_WALLET_FACTORY_V0_11042023_vyLkpGh",
   WALLET_IMP = "DEVX_WALLET_IMP_V0_11042023_AwPKF0R",
   SINGELTON_PAYMASTER = "DEVX_SINGLETON_PAYMASTER_V0_11042023_v6ISI9i",
+  ECDSA_REGISTRY_MODULE = "DEVX_ECDSA_REGISTRY_MODULE_V0_xxxxxxxxxxxxx",
+  MULTICHAIN_VALIDATOR_MODULE = "DEVX_MULTICHAIN_VALIDATOR_MODULE_V0_xxxxxxxxxxxxx",
+  PASSKEY_MODULE = "DEVX_PASSKEY_MODULE_V0_xxxxxxxxxxxxx",
+  SESSION_KEY_MANAGER_MODULE = "DEVX_SESSION_KEY_MANAGER_MODULE_V0_xxxxxxxxxxxxx",
+  ERC20_SESSION_VALIDATION_MODULE = "DEVX_ERC20_SESSION_VALIDATION_MODULE_V0_xxxxxxxxxxxxx",
+  SMART_CONTRACT_OWNERSHIP_REGISTRY_MODULE = "DEVX_SMART_CONTRACT_OWNERSHIP_REGISTRY_MODULE_V0_xxxxxxxxxxxxx",
 }
 
+type StakingConfig = {
+  unstakeDelayInSec: number;
+  stakeInWei: BigNumber;
+};
+
+export const factoryStakeConfig: Record<number, StakingConfig> = {
+  80001: {
+    unstakeDelayInSec: 60 * 60 * 24, // 1 Day
+    stakeInWei: ethers.utils.parseEther("0.001"),
```

Comment:
lets add staking config for all chains for approx 100 USD amount.

Reply:
added